Source/WarpX.cpp

```diff
@@ -880,17 +880,16 @@ WarpX::ReadParameters ()
         v_particle_pml = v_particle_pml * PhysConst::c;
 
         // Default values of WarpX::do_pml_dive_cleaning and WarpX::do_pml_divb_cleaning:
-        // false for FDTD solver, true for PSATD solver.
-        if (electromagnetic_solver_id != ElectromagneticSolverAlgo::PSATD)
-        {
-            do_pml_dive_cleaning = false;
-            do_pml_divb_cleaning = false;
-        }
-        else
+        // true for Cartesian PSATD solver, false otherwise
+        do_pml_dive_cleaning = false;
+        do_pml_divb_cleaning = false;
+#ifndef WARPX_DIM_RZ
+        if (electromagnetic_solver_id == ElectromagneticSolverAlgo::PSATD)
         {
             do_pml_dive_cleaning = true;
             do_pml_divb_cleaning = true;
         }
+#endif
```

Comment:
Just below this code block there's another couple of lines that read
```cpp
 if (do_dive_cleaning) do_pml_dive_cleaning = true;
```
and similar for `do_pml_divb_cleaning`.

I think these should also be executed only in Cartesian geometry.
I believe this should be enough to fix the CI tests that currently fail.